code/game/gamemodes/cult/cult_structures.dm

```diff
@@ -207,19 +207,20 @@ var/list/blacklisted_pylon_turfs = typecacheof(list(
 		user << "<span class='cultitalic'>The magic in [src] is weak, it will be ready to use again in [getETA()].</span>"
 		return
 	var/choice = alert(user,"You flip through the black pages of the archives...",,"Supply Talisman","Shuttle Curse","Veil Walker Set")
-	var/pickedtype
+	var/pickedtype[0]
```

Comment:
This should probably be `var/pickedtype = list()`; I'm not sure there's a difference, but this is a rather rare way to do it.